modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,159 @@
+
+Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Redpanda Dedicated clusters for private networking.
+
+=== Limitations
+
+Each Serverless cluster has the following limits:
+
+* Ingress: up to 10 MBps, 0.5 MBps guaranteed
+* Egress: up to 30 MBps, 1.5 MBps guaranteed
+* Partitions: 100
+* Max message size: 20 MB
+* Max retention: unlimited
+* Storage: unlimited
+* Users: 30
+* ACLs: 120
+* Consumer groups: 200
+
+=== Serverless vs Dedicated and BYOC
+
+Redpanda Serverless is a good fit for the following:
+
+* Starter and growning workloads
+* Spiky workloads (dev environments, systems that only occasionally get busy, or workloads that come and go)
+* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
+
+Because a Serverless cluster is created in under 1 second, you can surface it in your applications (for example, for tenant isolation).
+
+If necessary, workloads can be migrated from Redpanda Serverless to Redpanda Dedicated or BYOC clusters. Consider Dedicated or BYOC for the following:
+
+* Control over cluster settings
+* Control over the deployment
+* Private networking
+* Workloads with consistently-high throughput
+
+== Free trial 
+
+Start using Redpanda Serverless Beta by joining the free trial! To ask questions, give feedback, or extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+
+=== Prerequisites
+
+. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a Beta trial^] of Redpanda Serverless.
+
+. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
+
+=== Credits and limits
+
+Each Beta trial supports five Serverless clusters. Redpanda issues 100 free credits, which is enough to run Redpanda with reasonable throughput. These credits expire in 14 days. Usage is based on the following:
+
+* Ingress: 0.1 credit per GB in (writes)
+* Egress: 0.1 credit per GB out (reads)
+* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hr	
+* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/mth
+
+After either the 14 day trial expires or the 100 credits expire, the clusters move into a suspended state, and you won't be able to access your data in the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. 
+
+=== Get started
+
+For the Beta release, you'll receive an email to activate your Serverless trial. 
+
+. In your Welcome email, click to activate your cloud account.
++
+Redpanda prepares a Redpanda Cloud account for you with a "Hello, World!" Serverless cluster in AWS.
```

Comment:
Is the Serverless option accessible in a different part of the cloud UI? That is, if I'm a cloud user will I be able to tell that I'm interacting with specifically a Serverless element vs BYOC vs dedicated? And is the Serverless trial open to current Cloud customers?

Reply:
I believe the option to create a Serverless cluster is only visible when the feature flag is turned on (for users accepted into the trial). Users can see what type of cluster they have on the Overview page, and features that are not enabled yet in Serverless (like Schema Registry) just don't display for that cluster. 

The Serverless trial is open to current customers, but limited to a certain number, maybe 200. So it's not just a free-for-all.

Reply:
@kbatuigas @micheleRP 

- Serverless is just another cluster type. You select Serverless cluster type when creating the cluster in the UI.
- Serverless option is disabled by default. This stays while Serverless is in beta. We enable it for people who are Serverless beta users. We can selectively enable it for selected accounts. But it is disabled by default.